common/src/sql/dbinit.sql

```diff
@@ -1109,16 +1109,7 @@ CREATE TABLE omicron.public.ip_pool (
     time_modified TIMESTAMPTZ NOT NULL,
     time_deleted TIMESTAMPTZ,
 
-    /* Optional ID of the project for which this pool is reserved. */
-    project_id UUID,
-
-    /*
-     * Optional rack ID, indicating this is a reserved pool for internal
-     * services on a specific rack.
-     * TODO(https://github.com/oxidecomputer/omicron/issues/1276): This
-     * should probably point to an AZ or fleet, not a rack.
-     */
-    rack_id UUID,
+    internal_only BOOL NOT NULL,
```

Comment:
Minor bikeshedding here, no action required unless you feel like it. 

This could just be called `internal` as _only_ seems to me to be implied. 

It'd probably be good to copy the comment from the datastore struct or just note to check that file. Do `@see` comments work in rust? Maybe that's just a jsdoc thing. Again, not required. Just a nice to have.

Reply:
Done in b4dd0b1d7d16b92a54c9bb872401712334281d5d

I dunno about the linked comments, but I included a summary here.